data/tools/unit_tree/html_output.py

```diff
@@ -1235,6 +1235,12 @@ def clean_uval(name):
                         else:
                             error_message("Warning: Weapon special %s has no name for %s.\n" %
                                           (special.name.decode("utf8"), uid))
+                accuracy = attack.get_text_val("accuracy", default=0)
```

Comment:
I started thinking how default=0 instead of default="0" means that [attack]accuracy=0 would be displayed in html as accuracy 0%.

If someone has strong opinion it is useful to say.

Reply:
changed to "0"